package.json

```diff
@@ -1,18 +1,40 @@
 {
   "private": true,
+  "name": "ids-enterprise-ng",
+  "slug": "ids-enterprise-ng",
+  "version": "4.8.0-dev",
+  "description": "Infor Design System (IDS) Enterprise Angular Components",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/infor-design/enterprise-ng.git"
+  },
+  "keywords": [
+    "infor design system",
+    "infor design system enterprise angular"
+  ],
+  "author": "Hook & Loop",
+  "license": "Apache 2.0",
+  "bugs": {
+    "url": "https://github.com/infor-design/enterprise-ng/issues"
+  },
+  "homepage": "https://github.com/infor-design/enterprise-ng",
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
     "build": "ng build",
     "test": "ng test --browsers=PhantomJS --code-coverage",
     "lint": "ng lint",
     "e2e": "ng e2e",
-    "publish-setup": "cd ./publish && npm run prepublishOnly && cd ../",
+    "prepublishOnly": "npx gulp publish",
     "publish-dev": "npm publish publish/ --tag dev",
```

Comment:
We don’t need these now?

Reply:
Remember the green means something was added. The red means something was removed. 

I removed the cmd (publish-setup) that calls a cmd (prepublishOnly) from the now nonexistent publish/package.json, and we now have that cmd (prepublishOnly) available directly.